src/domain/reaction/ReactionData.ts

```diff
@@ -0,0 +1,32 @@
+
+export default class ReactionData
+{
+    #id: string;
+    #creatorId: string;
+
+    #comicId: string | undefined;
+    #commentId: string | undefined;
+
+    #ratingCount: number;
```

Comment:
The documentation is not updated with this new name

Reply:
As discussed, the purpose of the data model is for the initial design only. We've chosen to make the code the source of truth after the design phase to prevent additional overhead.